### PR TITLE
Handle all networks and switch between them from settings

### LINKED ISSUE #449: Settings: Support switching to Testnet
This needs to change internal address prefix and the explorer combos we use

### PR DESCRIPTION
**Pull request by Janaka-Steph**
Take initialState from /redux/config.ts and from there persisted Redux state is the source of truth.
Store new `defaultProvider` and `network`.

From settings, switching network will update: 
- network
- providers
  - only adding new ones but not deleting old ones
- defaultProvider
- explorer endpoints 
  - if testnet, testnet blockstream endpoints are set. User can then adjust from explorer settings
  - Mempool don't have Liquid testnet endpoints yet. Using Blockstream ones in the meantime

It closes #449 

Please review @tiero @louisinger

### CODE REVIEW

File: src/pages/Explorers/index.tsx [hunk 1]
```diff
@@ -59,17 +62,26 @@ const Explorers: React.FC<ExplorersProps> = ({
...
-      dispatch(setExplorerLiquidUI('https://liquid.network'));
+      dispatch(setExplorerLiquidAPI(mempoolExplorerEndpoints.liquid.explorerLiquidAPI));
+      dispatch(setExplorerLiquidUI(mempoolExplorerEndpoints.liquid.explorerLiquidUI));
+      dispatch(setExplorerBitcoinAPI(mempoolExplorerEndpoints.liquid.explorerBitcoinAPI));
+      dispatch(setExplorerBitcoinUI(mempoolExplorerEndpoints.liquid.explorerBitcoinUI));
+    } else if (value === 'mempool-testnet') {
+      dispatch(setExplorerLiquidAPI(mempoolExplorerEndpoints.testnet.explorerLiquidAPI));
+      dispatch(setExplorerLiquidUI(mempoolExplorerEndpoints.testnet.explorerLiquidUI));
+      dispatch(setExplorerBitcoinAPI(mempoolExplorerEndpoints.testnet.explorerBitcoinAPI));
+      dispatch(setExplorerBitcoinUI(mempoolExplorerEndpoints.testnet.explorerBitcoinUI));
```
Comment (tiero) [inline src/pages/Explorers/index.tsx:84]: I would rather remove the option of Elements Testnet instead

File: README.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+
+📱 Mobile app for making trades on TDEX
 
 ## ⬇️ Install
 
 * Download iOS from [App Store](https://apps.apple.com/app/truedex-trading-unleashed/id1545948177)
-* Download Android from [Play Store](https://play.google.com/store/apps/details?id=io.sevenlabs.app) or install the [APK from Github Releases](https://github.com/TDex-network/tdex-app/releases)
-
+* Download Android from [Play Store](https://play.google.com/store/apps/details?id=io.sevenlabs.app) or install
+  the [APK from Github Releases](https://github.com/TDex-network/tdex-app/releases)
```
Comment (tiero) [inline README.md:9]:
```suggestion
  the [APK from Github Releases](https://github.com/tdex-network/tdex-app/releases)
```

File: src/pages/Settings/Network.tsx [hunk 3]
```diff
@@ -75,6 +76,9 @@ const Network = (): JSX.Element => {
       // Refresh providers
       await refreshProviders(providers, network, dispatch);
       dispatch(clearAddresses());
+      dispatch(resetUtxos());
```
Comment (tiero) [inline src/pages/Settings/Network.tsx:79]: We should prevent to dispatch those, if one is already been dispatched, to save honerous requests.

In Marina we prevented switching networks if an update is not complete.

Comment (tiero) [reply src/pages/Settings/Network.tsx:79]: Altough we can do things in background in parallel

Comment (Janaka-Steph) [reply src/pages/Settings/Network.tsx:83]: Checking isFetchingUtxos

**Comment by tiero** [comment]
solve conflicts @Janaka-Steph

**Comment by tiero** [comment]
after that, LGTM

**Comment by wiz** [comment]
sorry for the delay, https://liquid.network/testnet is live for your testing now

**Pull request merged**